shemaeric/Politico
# 163897614 chore footer

#### What does this PR do?
This PR adds the Footer on the sign in, signup and reset password pages and fix the responsive for the pages
#### Description of Task to be completed?
Add the footer on mentioned pages and fix the responsive to mobile
#### How should this be manually tested?
This can be manually tested by cloning this repo and do git checkout 163897614-chore-footer and open the page on the web browser to see the changes.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#163897614](https://www.pivotaltracker.com/n/projects/2240082)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/36806576/52629618-c3802880-2ec2-11e9-95b8-205dd21d57b7.png)

#### Questions:
N/A
